pkg/scheduler/extender/extender.go

```diff
@@ -589,27 +609,69 @@ func nodePrioritize(nodeMapping map[string][]volcrd.Volume) (map[string]int64, i
 	return nrank, maxCount
 }
 
-// scNameStorageTypeMapping reads k8s storage class resources and collect map with key storage class name
-// and value .parameters.storageType for that sc, collect only sc that have provisioner e.provisioner
-func (e *Extender) scNameStorageTypeMapping(ctx context.Context) (map[string]string, error) {
-	scs := storageV1.StorageClassList{}
+func generateEphemeralVolumeName(podName, volumeName string) string {
```

Comment:
this can be just `fmt.Sprintf(...)`

Reply:
We have already discussed it later. I wished to underline here that we don't use a common name for inline volumes. Meaningful function name gives more info to the developer